crates/ruff_python_formatter/src/other/arguments.rs

```diff
@@ -54,13 +58,26 @@ impl FormatNodeRule<Arguments> for FormatArguments {
                     };
                 }
                 args => {
-                    joiner
-                        .entries(
-                            // We have the parentheses from the call so the item never need any
-                            args.iter()
-                                .map(|arg| (arg, arg.format().with_options(Parentheses::Preserve))),
-                        )
-                        .nodes(item.keywords.iter());
+                    // Consider e.g. `f(*args, a=b, *args2, **kwargs)`, where args (`*args`,
```

Comment:
Nit: It may be worth explaining that the args and kwargs are formatted in their appearance in source and that the code here is a merge sort.